### DISCUSSION TITLE
Current queue as an entry in `Playlists`

### DISCUSSION THREAD

**Discussion by bilogic** [navidrome/navidrome#2560] [Ideas]
https://github.com/navidrome/navidrome/issues/600

I was looking up and down for the current queue to eventually realize it was in the "external player".
Possible to have current queue as one of the `Playlists` and have it "synced" with the external player upon any changes?

**Comment by ElHado** [comment]
I think a solution for this would solve my question, too: https://github.com/navidrome/navidrome/discussions/2550

**Comment by bilogic** [comment]
@ElHado Yes, it can. 

I now think that `Queue` should not be under `Playlists`, instead, it should be an entry in the sidebar and can be the first step towards Navidrome having its own player.
